app/components/FormFields/WarningMessage.js

```diff
@@ -1,19 +1,40 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
 
-const WarningMessage = ({ touched, error, warning }) =>
-  touched && (error || warning) ? (
-    <h6 className="text-danger">
-      {(error && <FormattedMessage {...error} />) ||
-        (warning && <FormattedMessage {...warning} />)}
-    </h6>
+import validationArrowIcon from 'svg/validationArrow';
+
+import Span from 'components/Span';
+import Icon from 'components/Icon';
+
+const WarningMessageStyled = styled.div`
+  display: flex;
+  align-items: center;
+  padding-top: 5px;
+  padding-bottom: 5px;
+  ${props => props.width && `flex-basis: ${props.width}px;`};
+`;
+
+const WarningMessage = ({ error, warning, className, isArrowed }) =>
+  error || warning ? (
+    <WarningMessageStyled className={className}>
```

Comment:
Is there a way to avoid the need to have `className` property on the component? I see it is being used only to pass `col-xl-6` class. Maybe have more specific property, e.g. `halfScreenWidth`?